build.gradle

```diff
@@ -672,6 +672,7 @@ project(':streams') {
     testCompile project(':core')
     testCompile project(':core').sourceSets.test.output
     testCompile libs.junit
+    testCompile libs.assertj
```

Comment:
You don't need this to use `assertThat` notation, `hamcrest` is included by default in JUnit and we can just use that?